### PR TITLE
create an 11ty structure

### PR DESCRIPTION
**Pull request by chunkydotdev**
## Description
Create a base structure for future 11ty pages, styling and templates

## Tasks
- [x] Initiate 11ty into the project
- [x] Add base template (base.njk)
- [x] Add css folder

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const CleanCSS = require('clean-css')
+module.exports = function (eleventyConfig) {
+  eleventyConfig.addFilter('cssmin', function (code) {
+    return new CleanCSS({}).minify(code).styles
+  })
+  eleventyConfig.addPassthroughCopy('./src/css')
+  eleventyConfig.addWatchTarget('./src/css')
```
Comment (warpfork) [inline .eleventy.js:3]: Oh wow, TIL this is a thing, distinct from addPassthroughCopy.  Heck.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "serve": "npx @11ty/eleventy --serve",
+    "build": "npx @11ty/eleventy"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/warpsys/zapps.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```
Comment (warpfork) [inline package.json]: This should be Apache2-or-MIT (or however that's formatted for this system, idk)

File: .eleventy.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const CleanCSS = require('clean-css')
+module.exports = function (eleventyConfig) {
+  eleventyConfig.addFilter('cssmin', function (code) {
+    return new CleanCSS({}).minify(code).styles
```
Comment (warpfork) [inline .eleventy.js]: What's this do?  Is it just a css minifier?

Do we need it?  I think the amount of css we have... It probably does not matter.  And I bet I'd rather have the milliseconds of built time back (and smaller dependency tree) unless this really pulls a lot of weight somehow.

File: package.json (hunk 2 above)
Comment (chunkydotdev) [reply package.json]: Nice catch! Which one do you preferr?

File: .eleventy.js (hunk 3 above)
Comment (chunkydotdev) [reply .eleventy.js]: That's true, ill drop it then

**Review by warpfork — APPROVED** [review]
Okay, I think that LGTM :D and it gets us started nicely.

**Pull request merged**